src/applications/edu-benefits/components/EducationWizard.jsx

```diff
@@ -384,31 +385,60 @@ export default class EducationWizard extends React.Component {
-                            target="_blank"
-                            onClick={() =>
-                              recordEvent({
-                                event: 'edu-navigation',
-                                'edu-action': 'see-approved-stem-programs',
-                              })
-                            }
-                          >
-                            See eligible degree programs
-                          </a>
-                        </li>
-                      </ul>
+                      {environment.isProduction() ? (
+                        <ul className="circle-bullet ul-styling vads-u-margin-bottom--neg1">
+                          <li className="li-styling">
+                            You're enrolled in a bachelor’s degree program for
+                            science, technology, engineering, or math (STEM),{' '}
+                            <b>or</b>
+                          </li>{' '}
+                          <li>
+                            {' '}
+                            You've already earned a STEM bachelor’s degree and
+                            are pursuing a teaching certification.{' '}
+                            <a
+                              href="https://benefits.va.gov/gibill/docs/fgib/STEM_Program_List.pdf"
+                              rel="noopener noreferrer"
+                              target="_blank"
+                              onClick={() =>
+                                recordEvent({
+                                  event: 'edu-navigation',
+                                  'edu-action': 'see-approved-stem-programs',
+                                })
+                              }
+                            >
+                              See eligible degree programs
+                            </a>
+                          </li>
+                        </ul>
+                      ) : (
+                        <ul className="circle-bullet ul-styling vads-u-margin-bottom--neg1">
+                          <li className="li-styling">
+                            You're enrolled in a bachelor’s degree program for
+                            science, technology, engineering, or math (STEM),{' '}
+                            <b>or</b>
+                          </li>{' '}
+                          <li className="li-styling">
+                            You've already earned a STEM bachelor’s degree and
+                            are working toward a teaching certification,{' '}
+                            <b>or</b>
+                          </li>{' '}
+                          <li className="li-styling">
+                            {' '}
+                            You've already earned a STEM bachelor's or graduate
+                            degree and are pursuing a covered clinical training
+                            program for health care professionals. <br />
+                            <a
```

Comment:
Missing onClick

Reply:
https://github.com/department-of-veterans-affairs/vets-website/pull/17646/files#r655565914